app/build.gradle

```diff
@@ -19,7 +18,7 @@ ext.getLocalProperty = { propertyName ->
 
 android {
     compileSdkVersion 26
-    buildToolsVersion "26.0.2"
+    buildToolsVersion "27.0.0"
```

Comment:
I think we should be compiling SDK 27 if we're using the new build tools

Reply:
i will revert it to 26. I would rather not target beta versions